home/tests/test_models.py

```diff
@@ -953,3 +954,32 @@ def test_template_create_failed(self):
         wat_from_db = WhatsAppTemplate.objects.last()
         assert wat_from_db.submission_status == "FAILED"
         assert "Error" in wat_from_db.submission_result
+
+
+class IntegerQuestionBlockTests(TestCase):
+    def create_min_max_value(
+        self,
+        min,
+        max,
+    ):
+        return {
+            "min": min,
+            "max": max,
+        }
```

Comment:
This looks like more code than just sending a dict directly to clean(), any reason for this?

Reply:
No reason in particular. I just felt it was cleaner to do this with a function instead of sending the dict directly as
IntegerQuestionBlock().clean({
"min": 40,
"max": 50
}